scripts/release/deploy_staging_jars.sh

```diff
@@ -75,6 +75,7 @@ declare -a ALL_VERSION_OPTS=(
 "-Dscala-2.12 -Dflink1.14 -Davro.version=1.10.0 -pl packaging/hudi-flink-bundle -am"
 "-Dscala-2.12 -Dflink1.15 -Davro.version=1.10.0 -pl packaging/hudi-flink-bundle -am"
 "-Dscala-2.12 -Dflink1.16 -Davro.version=1.10.0 -pl packaging/hudi-flink-bundle -am"
+"-Dscala-2.12 -Dflink1.17 -Davro.version=1.10.0 -pl packaging/hudi-flink-bundle -am"
```

Comment:
I believe flink 1.17 is using avro 1.11 https://github.com/apache/flink/blob/release-1.17/pom.xml#L144

Reply:
Yeah, we need to fix the `flink.avro.version` as 1.11 for the flink1.17 profile specially.

Reply:
I have fixed this in new commit.